src/util/timeSeries.js

```diff
@@ -0,0 +1,18 @@
+const { computeDate, getPureDate } = require('./time')
+const ms = require('ms')
+
+const retentionLogDuration = '31 days'
+
+function getRetentionLimitDate () {
+  // round to pure date (0h0m0s) to have a relative stable value to display if there is any error value
```

Comment:
The idea was to have a retention limit date that remains the same for one-day duration, instead of subtracting 30 days duration from the current time (which would return a different value each time).
I'll update the comment as:
`round to pure date (0h0m0s) to return the same value for one-day duration...`

Reply:
I was just fixing your typo `relative stable` -> `relatively stable`.